app/src/main/java/one/mixin/android/util/PubkeyValidator.kt

```diff
@@ -0,0 +1,11 @@
+package one.mixin.android.util
+
+import cafe.cryptography.curve25519.CompressedEdwardsY
+
+fun verifyPubkey(bytes: ByteArray): Boolean {
+    return try {
+        return !CompressedEdwardsY(bytes).decompress().isSmallOrder
```

Comment:
Should validate the bytes length first.

Reply:
Validated [here](https://github.com/cryptography-cafe/curve25519-elisabeth/blob/master/src/main/java/cafe/cryptography/curve25519/CompressedEdwardsY.java#L37)